modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -26,26 +29,35 @@ metadata:
 spec:
   namespace: netobserv
   deploymentModel: Direct
-  agent:
-    type: eBPF
-    ebpf:
-      flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+agent:
+  type: eBPF
+  ebpf:
```

Comment:
```suggestion
     ebpf:
```

Reply:
you sure? I don't see an indentation issue, `ebpf` must be at the same level as `type`

Reply:
oh yes, my bad. Ignore this